Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp

```diff
@@ -162,14 +169,56 @@ namespace O3DE::ProjectManager
         // Simulate repo being entered and UI enabling
         connect(
             m_repoPath->lineEdit(), &QLineEdit::textEdited,
-            [this](const QString& text)
+            [this](const QString& /*text*/)
```

Comment:
you probably need to mark this `[[maybe_unused]]` pretty sure this will give a warning when run in AR